Sources/Sentry/Public/SentryOptions.h

```diff
@@ -157,6 +157,12 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL sendDefaultPii;
 
+/**
+ * When enabled, the SDK tracks UI performance automatically for UIViewController subclasses.
+ * The default is <code>YES</code>.
+ */
+@property (nonatomic, assign) BOOL enableAutoUIPerformanceTracking;
```

Comment:
m: we could align the names here, like its on Android `enableAutoActivityLifecycleTracing`
so ideally `enableAutoViewControllerLifecycleTracing`?

`UIPerformance` is very generic if its only about view controller lifecycle

Reply:
UIViewController are the core of ui in cocoa but is not the only thing being monitored here. A lot of work is done in UIView, and we will monitored other tasks that have an impact on the ui like http requests, database requests. 

I don't think using this specific naming is the answer, another reason is the fact that UIKit is not the only framework for ui in Apple environment, and SwiftUI (not being monitored right now) will probably be the default one.

Reply:
the problem is that a user maybe don't want to monitor them all but a subset of it, eg, I want to trace UI lifecycle, but I don't want to trace HTTP requests, or the other way around, we always try to make it customized that would fit most of the common use cases, but enabling them all or disabling them all isn't the best idea, happy to discuss.

Reply:
You are right, we need to separate these options. 
But I still think the name should be platform agnostic.

Reply:
Created an issue for this https://github.com/getsentry/sentry-cocoa/issues/1135.